tests/cupyx_tests/scipy_tests/sparse_tests/test_base.py

```diff
@@ -81,4 +82,4 @@ def test_asformat_none(self, xp, sp):
     @testing.numpy_cupy_equal(sp_name='sp')
     def test_maxprint(self, xp, sp):
         s = self.dummy_class(sp)(maxprint=30)
-        return s.getmaxprint()
+        return s.maxprint
```

Comment:
note: `getmaxprint` was deprecated in SciPy 1.11.

Reply:
Note for later reference: this is deprecated in sparray but not in spmatrix, so we don't have to add DeprecationWarning for getmaxprint.

https://docs.scipy.org/doc/scipy/release/1.11.0-notes.html#deprecated-features